### PR TITLE
`omni-executor` basic CI checks

### PR DESCRIPTION
**Pull request by kziemianek**
Adds checks for:

* fmt
* clippy
* build
* test

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -570,6 +586,21 @@ jobs:
         if: failure()
         uses: andymckay/cancel-action@0.5
 
+  omni-executor-check:
```
Comment (Kailai-Wang) [inline .github/workflows/ci.yml:589]: Would you check clippy here too?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -147,6 +157,7 @@ jobs:
           echo "rebuild_parachain=$rebuild_parachain" | tee -a $GITHUB_OUTPUT
           echo "rebuild_identity=$rebuild_identity" | tee -a $GITHUB_OUTPUT
           echo "rebuild_bitacross=$rebuild_bitacross" | tee -a $GITHUB_OUTPUT
+          echo "check_omni_executor=$check_omni_executor" | tee -a $GITHUB_OUTPUT
```
Comment (Kailai-Wang) [inline .github/workflows/ci.yml:160]: this variable seems unused in the CI

Comment (kziemianek) [reply .github/workflows/ci.yml:160]: it's now used

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kziemianek) [reply .github/workflows/ci.yml:589]: added

**Review by Kailai-Wang — APPROVED** [review]

**Pull request merged**